app/views/admin/AnalyticsView.coffee

```diff
@@ -16,21 +16,9 @@ module.exports = class AnalyticsView extends RootView
   lineColors: ['red', 'blue', 'green', 'purple', 'goldenrod', 'brown', 'darkcyan']
   minSchoolCount: 20
 
-  constructor: (options) ->
-    super options
+  initialize: ->
     @loadData()
 
-  getRenderData: ->
-    context = super()
-    context.activeClasses = @activeClasses ? []
-    context.activeClassGroups = @activeClassGroups ? {}
-    context.activeUsers = @activeUsers ? []
-    context.revenue = @revenue ? []
-    context.revenueGroups = @revenueGroups ? {}
-    context.dayEnrollmentsMap = @dayEnrollmentsMap ? {}
-    context.enrollmentDays = @enrollmentDays ? []
```

Comment:
This code not only constructed the context, it also provided default values. Run with these changes, the view breaks because they try to iterate on `undefined`s. I suggest moving the default, empty value setting to `initialize`, so that this view always has something in those properties.